config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_sqlinstances.sql.cnrm.cloud.google.com.yaml

```diff
@@ -495,14 +487,15 @@ spec:
                           - external
                         properties:
                           external:
-                            description: 'Allowed value: The `selfLink` field of a
-                              `ComputeNetwork` resource.'
+                            description: The compute network selflink of form "projects/<project>/global/networks/<network>",
```

Comment:
Can we figure out a consistent way to describe the value of the external field? In the fields above you mentioned it is `The [KindName] selfLink, when not managed by KCC.` Can we be consistent about:
1. How do we plan to format the text?
2. When should we mention the format or other information?

Would be great to have a generator/enforcer for this in the future.

Reply:
fixed